nipype/interfaces/fsl/utils.py

```diff
@@ -1700,7 +1700,28 @@ def _gen_filename(self, name):
             return self._list_outputs()['out_file']
         return None
 
+      
+class FslOrientInputSpec(FSLCommandInputSpec):
+    in_file = File(exists=True, desc='input file', argstr='%s', position=2, mandatory=True)
+    main_option = traits.Str(desc='main option', argstr='-%s', position=0, mandatory=True)
+    code = traits.Int(argstr='%d', desc='code for setsformcode', position=1)
+
+    
+class FslOrientOutputSpec(TraitedSpec):
+    out_file = File(desc = "out file", exists = True)
+
+    
+class FslOrient(FSLCommand):
```

Comment:
Please add a minimal usage example analogous to that seen in other interfaces in this module:

https://github.com/nipy/nipype/blob/e722d6170593583f16ddfcb95473e5d30b5f1d7c/nipype/interfaces/fsl/utils.py#L220